website/docs/developer-guide/chaoscenter-developer-guide.md

```diff
@@ -6,6 +6,11 @@ sidebar_label: ChaosCenter Developer Guide
 
 ---
 
+## **Overview**
```

Comment:
Please remove the ‘Overview’ heading. Every document begins with an overview by default, so it’s not necessary to include it explicitly.